src/prob/opf_bf.jl

```diff
@@ -0,0 +1,51 @@
+export run_opf_bf, run_ac_opf_bf, run_dc_opf_bf
+
+""
+function run_ac_opf_bf(file, solver; kwargs...)
+    return run_opf_bf(file, ACPPowerModel, solver; kwargs...)
+end
+
+""
+function run_dc_opf_bf(file, solver; kwargs...)
+    return run_opf_bf(file, DCPPowerModel, solver; kwargs...)
+end
+
+""
+function run_opf_bf(file, model_constructor, solver; kwargs...)
+    return run_generic_model(file, model_constructor, solver, post_opf_bf; kwargs...)
+end
+
+""
+function post_opf_bf(pm::GenericPowerModel)
+    variable_voltage(pm)
+    variable_generation(pm)
+    variable_branch_flow(pm)
+    variable_dcline_flow(pm)
+
+    objective_min_fuel_cost(pm)
+
+    constraint_voltage(pm)
```

Comment:
It is not immediately clear to me if this is needed in problem definition.  It might help you to comment it out for now.

Reply:
ok, removed